bot/exts/evergreen/cowsay.py

```diff
@@ -0,0 +1,51 @@
+from contextlib import suppress
+from typing import Optional
+
+from cowsay import char_names, get_output_string
+from discord import HTTPException
+from discord.ext import commands
+
+from bot.bot import Bot
+
+# Creates a local copy of char_names to filter out unsupported characters.
+SUPPORTED_CHAR_NAMES = list(char_names)
+for element in ["dragon", "trex", "stegosaurus", "turkey", "ghostbusters", "turtle"]:
+    SUPPORTED_CHAR_NAMES.remove(element)
+
+
+class Cowsay(commands.Cog):
+    """Cog for the cowsay command."""
+
+    @commands.command(
+        aliases=["cow"],
+        help=f"""
+        Generates a cowsay string and sends it. The available characters for cowsay are:
+
+        {', '.join(SUPPORTED_CHAR_NAMES)}"""
+    )
+    async def cowsay(self, ctx: commands.Context, character: str = "Cow", *, text: Optional[str]) -> None:
+        """Builds a cowsay string and sends it to Discord."""
+        if not text:
+            text = f"I'm a {character.lower()}"
+        character = character.lower()
```

Comment:
You are calling `character.lower()` twice, L30 can be moved up to prevent that.
```suggestion
        character = character.lower()
        if not text:
            text = f"I'm a {character.lower()}"
```